ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -52,6 +52,16 @@ tests:
     test:
     - ref: openshift-qe-node-density
     workflow: rosa-aws-sts
+- as: perfscale-rosa-multiaz-ingress-perf-3nodes
```

Comment:
Since this is a cron, can you please add "periodic" at the end of the test name? You can see other examples. Other than that PR lgtm

Reply:
Sure, updated, ptal